pilot/pkg/model/telemetry.go

```diff
@@ -362,7 +364,7 @@ func (t *Telemetries) telemetryFilters(proxy *Proxy, class networking.ListenerCl
 	// First, take all the metrics configs and transform them into a normalized form
 	tmm := mergeMetrics(c.Metrics, t.meshConfig)
 	// Additionally, fetch relevant access logging configurations
-	tml := mergeLogs(c.Logging, t.meshConfig)
+	tml, _ := mergeLogs(c.Logging, t.meshConfig)
```

Comment:
do we need filters here? Seems like we should support them, add a TODO, or a comment explaining why we do not need to.

I think TODO since SD does not support it yet but will in the future, but I am not certain.. @douglas-reid ?

Reply:
We will need filters for SD. There is an open PR (https://github.com/istio/proxy/pull/3630) to add that functionality.

Reply:
updated with a TODO